algorithm/cgal/cgal.cc

```diff
@@ -3549,6 +3549,61 @@ class Geometry {
 
   const Surface_mesh* getInputMesh(int nth) { return input_meshes_[nth]; }
 
+  std::string serializeMesh(const Surface_mesh& mesh) {
+    // CHECK: We assume the mesh is compact.
```

Comment:
What does it mean for a mesh to be compact?

Is it the same as being demeshed?

Reply:
Ah, meshes may contain deleted elements.

By compact I mean that the deleted elements have been removed, so there are no gaps in the vertex indices, etc.